### PR TITLE
add screen shader in 3d

### PR DESCRIPTION
**Pull request by ywaby**
add screen shader 3d demo

### CODE REVIEW

**Comment by Vega-KH** [comment]
Good addition.  Screen shaders have interesting effects on 3D scenes. 

Unrelated to this PR, but it would be nice to have a demo project with some 3D shaders as well.

**Comment by ywaby** [comment]
It can used to PostProcess effect

File: 2d/screen_space_shaders/MeshInstance.gd [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	pass
+
+
+var model = null
+const SPEED = 40
+
+
+
+func _process(delta):
+	rotation_degrees.y += delta * SPEED
```
Comment (aaronfranke) [inline 2d/screen_space_shaders/MeshInstance.gd]: Please ensure that there are newlines at the end of each file.

File: 2d/screen_space_shaders/MeshInstance.gd [hunk 2]
```diff
@@ -0,0 +1,19 @@
+extends MeshInstance
+
+# class member variables go here, for example:
+# var a = 2
+# var b = "textvar"
+
+func _ready():
+	# Called every time the node is added to the scene.
+	# Initialization here
+	pass
```
Comment (aaronfranke) [inline 2d/screen_space_shaders/MeshInstance.gd]: Remove unused functions and default comments.

**Comment by ywaby** [comment]
I think use viewport for custom post process shader  is simple.
reference to godot doc https://docs.godotengine.org/en/latest/tutorials/viewports/custom_postprocessing.html

**Comment by aaronfranke** [comment]
Unfortunately, this PR is just too old, and has not been updated with the changes that were requested. I'm also not quite sure if it would be desired to begin with. I'll close this PR, but feel free to discuss things below, or make a new one.

**Pull request closed without merging**